server/auth/auth_oauth.py

```diff
@@ -227,11 +229,12 @@ def get_jwt_payload(self):
                 )
                 return payload
 
-            except jwt.JWTError as e:
+            except JWTError as e:
                 raise AuthenticationError(f"invalid signature: {str(e)}")
-            except jwt.ExpiredSignatureError as e:
-                raise AuthenticationError(f"token expired: {str(e)}")
-            except jwt.JWTClaimsError as e:
+            except ExpiredSignatureError:
+                # TODO, handle expired sessions by refreshing the token
+                return None
```

Comment:
Why none instead of an AuthError?

Reply:
It's not an error for the token to be expired, it just means the user is no longer logged in.  
It is an error if the token is malformed, or does not have the right claims, audience, etc.

I have a PR in progress that will attempt to refresh the token in this case.